code/modules/jobs/jobs.dm

```diff
@@ -60,16 +60,17 @@ GLOBAL_LIST_INIT(nonhuman_positions, list(
 	"Cyborg",
 	ROLE_PAI))
 
+// include department colors for rendering the late join menu
 GLOBAL_LIST_INIT(exp_jobsmap, list(

```

Comment:
I don't feel this is appropriate global to use. 
New one would be more fitting.
(Or ideally we'd have departments defined with types somewhere but that's bigger refactor)

Reply:
Moved to a new global.